src/pages/Registration/Registration.tsx

```diff
@@ -88,9 +81,7 @@ const Registration = () => {
   };
 
   return (
-    <FormContainer
-      onClick={() => console.log('function to navigate to log in page')}
-    >
+    <FormContainer onClick={() => navigate('/login')} isLoginPage={false}>
```

Comment:
Please, add functionality to switch to Login page automatically after click on Join to Club.